### DISCUSSION TITLE
Thermostat range

### DISCUSSION THREAD

**Discussion by garyhooper** [jeffschubert/homebridge-daikin-oneplus#38] [General]
Hi Jeff,

Is there a reason why you limit the cooling range to 12-32C, and the heating range to 10-30C?  I ask because if I leave home for a trip during winter, I don't want to be heating the house to 10C...I'd rather lower it to 5C.

It's also not clear to me whether the Daikin One would observe extended ranges.

Thanks!

**Comment by jeffschubert** [comment]
Good question! I had to dig to make sure I understood again. (I wasn't the one to originally add the limits.)
In short, those are the ranges allowed by the Daikin app. It won't let you set a heat or cool target temp outside of those respective ranges.
Apple's HomeKit and Homebridge do allow a wider range, but to prevent causing issues with the system/thermostat/etc, I'd rather keep to the Daikin defined limitations.
